homeassistant/components/nest/strings.json

```diff
@@ -64,9 +64,9 @@
     }
   },
   "issues": {
-    "legacy_nest_deprecated": {
-      "title": "Legacy Works With Nest is being removed",
-      "description": "Legacy Works With Nest is being removed from Home Assistant.\n\nYou must take action to use the SDM API. Remove all `nest` configuration from `configuration.yaml` and restart Home Assistant, then see the Nest [integration instructions]({documentation_url}) for set up instructions and supported devices."
+    "legacy_nest_removed": {
+      "title": "Legacy Works With Nest has been removed",
+      "description": "Legacy Works With Nest has been removed from Home Assistant.\n\nYou must take action to use the SDM API. Remove all `nest` configuration from `configuration.yaml` and restart Home Assistant, then see the Nest [integration instructions]({documentation_url}) for set up instructions and supported devices."
```

Comment:
Should we tell them why it's been removed?  They may not know.

Reply:
You caught me being passive aggressive here. I was scheduled it for removal once already but reverted it because something in SDM broke... so I didn't want to blame it on the shutdown necessarily.  But since you asked, i'll update the wording...